config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
@sboeuf @grahamwhaley with this we will be switching to mainline QEMU from QEMU lite. So just qemu is fine. With the mainline qemu we are using the standard q35 machine type. It is just that we have not made it more configurable so that the q35 now allows disabling of a whole bunch of capabilities. So this is not "new q35". It is just mainline qemu that now support more disable options.

So here
```
-machine q35,accel=kvm,kernel_irqchip,nvdimm,nosmm,nosmbus,nosata,nopit,nofw
```
the no* options are mostly new.